core/src/main/java/org/elasticsearch/cluster/metadata/MetaDataIndexUpgradeService.java

```diff
@@ -150,7 +149,7 @@ private static boolean isSupportedVersion(IndexMetaData indexMetaData) {
                                     "index.translog.interval",
                                     "index.translog.sync_interval",
                                     "index.shard.inactive_time",
-                                    UnassignedInfo.INDEX_DELAYED_NODE_LEFT_TIMEOUT_SETTING));
+                                    "index.unassigned.node_left.delayed_timeout"));
```

Comment:
Should these be settings references? I'm not sure why they aren't but I'm sure there was a good reason but I don't know if its still true.